repositories.bzl

```diff
@@ -188,7 +188,7 @@ filegroup(
     native.new_git_repository(
         name = "mixerapi_git",
         build_file_content = BUILD,
-        commit = ISTIO_API,
+        commit = "b33fb6d1d44ae6e7252004c8916a702976cfb101",
```

Comment:
you should assign new sha  to ISTIO_API, not repleace it

Reply:
Temporary, until api is unfrozen and I can merge the corresponding PR.